corehq/apps/app_manager/tests/app_factory.py

```diff
@@ -97,6 +97,22 @@ def form_updates_case(form, case_type=None, parent_case_type=None, update=None):
             form.actions.load_update_cases.append(action)
 
     @staticmethod
+    def form_preloads_case(form, case_type=None, parent_case_type=None, preload=None):
+        if form.form_type == 'module_form':
```

Comment:
Would be nice to add support for advanced form too or at least error if an advanced for is passed in so that someone doesn't waste time trying to figure out what's going on.